examples/erlang/hello_world_server.erl

```diff
@@ -34,7 +34,7 @@ start() ->
         ]}
     ],
     case network_fsm:start(Config) of
```

Comment:
We depreciated `network_fsm` module a long time ago. Shouldn’t this be `network:start/1`?

Reply:
Exactly, thanks for pointhing this out.